public/editor/scripts/editor/js/fc/bramble-menus.js

```diff
@@ -88,11 +87,28 @@ define(function(require) {
       if(toggle) {
         $allowWSToggle.addClass("switch-enabled");
         bramble.enableWhiteSpace();
-        analytics.event("EnableWhiteSpaceIndicator");
+        analytics.event("EnableWhiteSpace");
       } else {
         $allowWSToggle.removeClass("switch-enabled");
         bramble.disableWhiteSpace();
-        analytics.event("DisableWhiteSpaceIndicator");
+        analytics.event("DisableWhiteSpace");
+    //set the AutoCloseTags toggle to reflect whether auto-close tags is enabled or disabled
+    if(bramble.getAutoCloseTags().whenClosing) {
+      $("#auto-tags-toggle").addClass("switch-enabled");
+    } else {
+      $("#auto-tags-toggle").removeClass("switch-enabled");
+    }
+
+    $("#auto-tags-toggle").click(function() {
```

Comment:
We use $("#auto-tags-toggle") here and then also on the next line. Since we know it will always be the same UI element (because we only have one), we can cache this:
```
var $autoTagsToggle = $("#auto-tags-toggle");
$autoTagsToggle .click(function() {
  // ... no need to refetch $autoTagsToggle
}
```